.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -9,6 +9,9 @@ assignees: ''
 
 > ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62
 
+**Related command**
+<!--- Please provide the related command with `az {command}`, azure-cli bot will analyse command and route to related team --->
```

Comment:
Please provide the related command with `az {command}`, so that we can quickly route to the related team for help.